packages/cloud-cognitive/src/internal/devtools.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright IBM Corp. 2021, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { paramCase } from 'param-case';
+
+const devtoolsAttribute = 'data-carbon-devtools-id';
```

Comment:
A thought: 'data-carbon-devtools-id' sounds fairly generic -- it doesn't specifically say C&CS, for example. So other libraries might conceivably adopt the same approach, and might wish to use the same attribute name.So perhaps we should include a prefix in the devtoolsId values we compute, i.e. xxx-about-modal rather than just about-modal. That would ensure that where libraries have components with the same name we can tell which is which. The 'xxx' could be our prefix, eg 'cip' or whatever we choose.